openassessment/xblock/static/js/spec/lms/oa_response.js

```diff
@@ -401,21 +401,36 @@ describe("OpenAssessment.ResponseView", function() {
 
     it("selects too large of a file", function() {
         spyOn(baseView, 'toggleActionError').andCallThrough();
-        var files = [{type: 'image/jpg', size: 6000000, name: 'huge-picture.jpg', data: ''}];
-        view.prepareUpload(files);
+        var files = [{type: 'image/jpeg', size: 6000000, name: 'huge-picture.jpg', data: ''}];
```

Comment:
haha ok you see how much I know about image types :grinning:  thanks for bearing with me :grimacing: